crates/ln-dlc-node/src/ln/mod.rs

```diff
@@ -16,12 +16,19 @@ pub(crate) use logger::TracingLogger;
 /// payment is small enough (< 1000 sats), opening the channel will
 /// fail unless we provide more outbound liquidity.
 ///
-/// This value is completely arbitrary at this stage. Eventually, we
-/// should, for example, let the payee decide how much inbound
-/// liquidity they desire, and charge them for it.
+/// This value defines the maximum channel amount between the coordinator and a user that opens a
+/// channel through an interceptable invoice. Channels that exceed this amount will be rejected.
+/// This value is completely arbitrary.
 ///
 /// This constant only applies to the coordinator.
-pub(crate) const JUST_IN_TIME_CHANNEL_OUTBOUND_LIQUIDITY_SAT: u64 = 200_000;
+pub(crate) const JUST_IN_TIME_CHANNEL_OUTBOUND_LIQUIDITY_SAT_MAX: u64 = 200_000;
+
+/// The multiplier to be used by the coordinator to define the just in time channel liquidity
+///
+/// The liquidity provided by the trader will be multiplied with this value to defined the channel
+/// value.
+/// See `JUST_IN_TIME_CHANNEL_OUTBOUND_LIQUIDITY_SAT_MAX` for the maximum channel value.
+pub(crate) const LIQUIDITY_MULTIPLIER: u64 = 2;
```

Comment:
nit: It would be nice if we could configure these values in the `coordinator-settings.toml`

Reply:
I think that would be a bigger change because the app also depends on these values. That means that the app initially has to fetch these from the coordinator. That can easily be added on top.

Reply:
in my review, I noted that they could become part of the `ChannelInfo` - this would be one potential way to achieve passing it to the app from the coordinator.

Reply:
Fair point - would you be OK if we schedule this in a follow up?